mandatory/3-function-output.js

```diff
@@ -1,14 +1,15 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getRandomNumber() {
   return Math.random() * 10;
-}
+} //will probably pick a random num (sorry Greg!) and multipley it by 10.
```

Comment:
Question: A random number from where to where?

Reply:
Hi Abdi, I think it should return a value between 0 and 9.
Thank you for the valuable advises mate.